server/src/test/java/org/elasticsearch/action/admin/indices/close/TransportVerifyShardBeforeCloseActionTests.java

```diff
@@ -180,20 +167,17 @@ public void testOperationFailsWithNoBlock() {
         IllegalStateException exception = expectThrows(IllegalStateException.class, this::executeOnPrimaryOrReplica);
         assertThat(exception.getMessage(),
             equalTo("Index shard " + indexShard.shardId() + " must be blocked by " + clusterBlock + " before closing"));
-        verify(indexShard, times(0)).flush(any(FlushRequest.class));
```

Comment:
I think it could make sense to verify that `prepareShardBeforeIndexClosing` is not called.